gemfirexd/shared/src/main/resources/com/pivotal/gemfirexd/internal/loc/toolsmessages.properties

```diff
@@ -545,6 +569,86 @@ FS_EXTRA_HELP=\
 \                 data and will act as an accessor JVM still having the\n\
 \                 capability to route queries to appropriate datastores and\n\
 \                 aggregating the results.\n
+#snappy specific message hides mcast port args, otherwise same as above
+FS_SNAPPY_EXTRA_HELP=\
+\   -sync         If true then wait for complete initialization of the server\n\
+\                 including DataDictionary, persistent tables, network server,\n\
+\                 else return when this node starts waiting for other members\n\
+\                 to join to sync persistent data (default false).\n\
+\   -heap-size    Sets both the initial heap and max heap sizes.\n\
+\                 Size has the same format as the -Xmx/-Xms JVM options.\n\
+\                 -heap-size is equivalent to specifying both -J-Xmx and -J-Xms.\n\
+\   -off-heap-size Size has the same format as the -Xmx/-Xms JVM options.\n\
```

Comment:
Change to memory-size as mentioned above. Also add something like "When set then all column tables use off-heap memory (aka DirectByteBuffer) to the specified total limit."